giddy-banana/package.json

```
@@ -0,0 +1,22 @@
+{
+  "name": "giddy-banana",
+  "version": "0.1.0",
+  "description": "The backend service of WikiOnFire",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/kpagacz/wiki-on-fire.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/kpagacz/wiki-on-fire/issues"
+  },
+  "homepage": "https://github.com/kpagacz/wiki-on-fire#readme",
+  "dependencies": {
+    "express": "^4.17.1"
```

```suggestion
    "express": "^4.17.1",
    "type": "module
```

I assume this is it?

---

"dependencies: {
...
},
"type": "module"

---

Done